Changed body background color

veekeys opened the pull request:
- Changed `<body>` background color from `$iui-color-background-1` to `$iui-color-background-2`.
- Tweaked fieldset background & border color to display better over background-2. *Discussed between @veekeys & @FlyersPh9.*

FlyersPh9 commented:
How did only 2 files change?  Wouldn't nearly every single test image change?

bentleyvk commented:
> How did only 2 files change? Wouldn't nearly every single test image change?

I think there is a bug in BackstopJS that it is not checking or have very high threshold for color differences.

veekeys commented:
> > How did only 2 files change? Wouldn't nearly every single test image change?
> 
> I think there is a bug in BackstopJS that it is not checking or have very high threshold for color differences.

Yeah, so after changing the background color, only single test failed..
If I look at the passed ones, obviously there is a color change:
![image](https://user-images.githubusercontent.com/81580355/122721427-52629400-d279-11eb-96cc-02ea92c751b8.png)

But that did not fail anywhere else.. So the only issue I have, all other images are going to be with wrong bacgkround color and maybe in future that could cause failures..
Ill try to play around to have all the test failing.

veekeys commented:
@FlyersPh9 
I have updated all images by deleting and running tests...
Mostly looks okeish I guess, but.. these components in particular did not look good on background-2
fieldset
fileupload
tags

So.. I am not sure, what we do.
- We could have background 1 in all the html files, just have background-2 on body in css applied.
- We can leave as I updated now (review please)
- We can do something else to make them look good on background-2.. So I am not sure, whats the intention. I know most of the components should appear on different background, but.. all this stuff is getting really confusing.

FlyersPh9 commented:
> fieldset

Fieldset I anticipate only being used above background 1.  But if we can change the fieldset to use background-4 instead of background-3 so that it displays better.
![Screen Shot 2021-07-07 at 11 41 13 AM](https://user-images.githubusercontent.com/849817/124789394-3cf79600-df18-11eb-92c5-f4f73efd2f5c.png)

> fileupload

When file upload is needed above background-2, I am thinking it would be done similar to how [iTwin Design Review](https://review.itwin.bentley.com/home) handled it.
![Screen Shot 2021-07-07 at 11 51 54 AM](https://user-images.githubusercontent.com/849817/124791071-bb086c80-df19-11eb-9cf5-17d24f86345d.png)

When used above background-1, no changes needed.
![Screen Shot 2021-07-07 at 11 52 46 AM](https://user-images.githubusercontent.com/849817/124791215-dbd0c200-df19-11eb-8b54-39d05acfaa7a.png)

> tags

For tags, we have the `.iui-visible` class which adds a little bit of padding and changes the background color to 2.  If they needed to use this over background 2, they could either remove `.iui-visible` or override it to use background 1.
![Screen Shot 2021-07-07 at 12 04 14 PM](https://user-images.githubusercontent.com/849817/124792900-74b40d00-df1b-11eb-9bd3-4c8a66a59d8a.png)

I'd really like to set some time aside soon to work on [some fake full page UIs](https://vfble.csb.app/) so we can see all the UI components in context of the UI.  I think it will help us make visual design decisions.

veekeys commented on src/style/global.scss:
```diff
@@ -4,7 +4,7 @@
 .iui-body {
   @include themed {
-    background-color: t(iui-color-background-1);
+    background-color: t(iui-color-background-2);
```
@FlyersPh9

FlyersPh9 reviewed (approved):
I made the adjustments that @veekeys and I spoke about on the phone to fieldset.  PR should be good to pass.

mayank99 commented:
Need to remove these html overrides

https://github.com/iTwin/iTwinUI/blob/c68f2508dbaf301255542ae19bbae2f73d824eec/backstop/tests/header.html#L58-L59
https://github.com/iTwin/iTwinUI/blob/c68f2508dbaf301255542ae19bbae2f73d824eec/backstop/tests/side-navigation.html#L25-L26
